Jenkinsfile

```diff
@@ -0,0 +1,53 @@
+import groovy.json.JsonOutput
+
+
+timestamps {
+  node('regular-memory-node') {
+    stage('checkout') {
+      scmInfo = checkout scm
+      println("${scmInfo}")
+      env.GIT_BRANCH = scmInfo.GIT_BRANCH
+      env.GIT_COMMIT = scmInfo.GIT_COMMIT
+    }
+    params = [
+      string(name: 'svn_revision', value: 'main'),
+      string(name: 'branch', value: 'main'),
+      string(name: 'client_git_commit', value: scmInfo.GIT_COMMIT),
+      string(name: 'client_git_branch', value: scmInfo.GIT_BRANCH),
+      string(name: 'TARGET_DOCKER_TEST_IMAGE', value: 'go-centos7-go1.21'),
```

Comment:
We still support older go version, shouldn't we use the oldest one?

Reply:
changed to 1.19